drucker/utils/__init__.py

```diff
@@ -93,3 +93,15 @@ class EvaluateDetail(NamedTuple):
     label: PredictLabel
     result: PredictResult
     is_correct: bool
+
+
+incoming_headers = [
+    'x-request-id', 'x-b3-traceid', 'x-b3-spanid', 'x-b3-parentspanid',
+    'x-b3-sampled', 'x-b3-flags', 'x-ot-span-context']
+
+def getForwardHeaders(incoming: list) -> list:
```

Comment:
To me, it seems better to use snake except grpc protocol methods.

Reply:
@yuki-mt Thank you.
This style comes from Istio official example.
https://istio.io/docs/tasks/telemetry/distributed-tracing/

Reply:
Oh, then it's ok. We should follow official example.

Reply:
Do we need to send back the metadata for tracing?
I thought so, but the sample application does not look to do so.

@keigohtr

Reply:
Istio adds these metadata and monitors them to calculate the processing time.
But I am not sure it works correctly.

Reply:
I will check it again how it works.